packages/reports/tests/integration/components/metric-config-test.js

```diff
@@ -190,6 +195,34 @@ module('Integration | Component | metric config', function(hooks) {
 
     //remove Param `Dollars(USD)`
     await clickItem('metricConfig', 'USD');
+
+    //enableRequestPreview feature flag on
+    config.navi.FEATURES.enableRequestPreview = true;
+
+    await render(TEMPLATE);
+
+    set(this, 'addParameterizedMetric', (metric, param) => {
+      assert.deepEqual(metric, MockMetric, 'The mock metric is passed to the action when enableRequestPreview is on');
+
+      assert.deepEqual(
+        param,
+        { currency: 'AMD' },
+        'The selected param is also passed to the action when enableRequestPreview is on'
+      );
+    });
+
+    set(this, 'removeParameterizedMetric', () =>
+      assert.notOk(true, 'removeParameterizedMetric is not called when enableRequestPreview is on')
+    );
+
+    await clickTrigger('.metric-config__dropdown-trigger');
+
+    await clickItem('metricConfig', 'AMD');
```

Comment:
Heads up, metric config now does searching by description, so peek [this commit](https://github.com/yahoo/navi/pull/722/commits/3059603e4d626323fb16d58ac7a2908bd19b95cf) before rebasing